### PR TITLE
Lexllanes

### PR DESCRIPTION
**Pull request by Lex-Llanes**

### CODE REVIEW

**Comment by botanical** [comment]
Great work!

**Review by botanical — COMMENTED** [review]
This is an example of a review comment

File: Pair-Routes/about.html [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <div>This is the aboutHTML</div>
```
Comment (botanical) [inline Pair-Routes/about.html:10]: You can leave a space between `about` and `HTML`

Comment (Lex-Llanes) [reply Pair-Routes/about.html:10]: Oh thats an amazing suggestion TY!

**Pull request merged**